das/state.go

```diff
@@ -6,11 +6,13 @@ import (
 
 // coordinatorState represents the current state of sampling
 type coordinatorState struct {
-	rangeSize uint64
+	genesisHeight uint
+	samplingRange uint // is the maximum amount of headers processed in one job.
```

Comment:
lets use `uint64` here to avoid multiple conversions on some operations.

Reply:
I will for the time being, and will also add a `TODO` to revise whether we should keep `uint64` for the variables it's being compared with.